net/upnp/src/www/services_upnp.php

```diff
@@ -139,6 +141,9 @@ function miniupnpd_validate_port($port)
     if (!empty($pconfig['upload']) && ($pconfig['upload'] <= 0 || !is_numeric($pconfig['upload']))) {
         $input_errors[] = gettext('You must specify a value greater than 0 in the \'Maximum Upload Speed\' field');
     }
+    if (!empty($pconfig['num_permuser'] && ($pconfig['num_permuser'] < 1 || !is_numeric($pconfig['num_permuser'])))) {
```

Comment:
shouldn't it check for numeric first before comparing numeric? :)

Reply:
That . . . would probably make more sense, yes.  I'll change it.  Though, the lines above (for `download` and `upload`) have the same issue.  Would it make sense to change those as well in this same PR?

Reply:
sure, that would be good to clean up, nice spot :)

Reply:
OK, that's in now.  :^)